oggm/core/massbalance.py

```diff
@@ -238,6 +238,7 @@ def __init__(self, gdir, mu_star=None, bias=None, prcp_fac=None,
             ny, r = divmod(len(time), 12)
             if r != 0:
                 raise ValueError('Climate data should be N full years')
```

Comment:
Maybe it is a good moment to also include checking if the climate data actually start in January.

Reply:
Actually, we assume the that the data starts in October (because of the hydrological year)

Reply:
Somehow I mixed things up, but you are right.